pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -321,6 +325,12 @@ func Run() {
 				log.Infof("Verifier reported restarted\n")
 			}
 
+		case change := <-subImageStatus.C:
```

Comment:
subImageStatus.MsgChan()

Reply:
What I have should fail to compile once I pull in origin/master, right?

Reply:
Fixed